engine/app/components/citizens_advice_components/header.html.haml

```diff
@@ -0,0 +1,43 @@
+%header.cads-header
+  .cads-grid-container
+    .cads-grid-row
+      - if skip_links.present?
+        %nav.cads-skip-links
+          - skip_links.each do |link|
+            = link_to link.title, link.url, class: "cads-skip-links__link"
+
+      .cads-grid-col-md-2.cads-header__logo-row
+        = logo
+        = toggle_button
+
+      .cads-grid-col-md-10.cads-header__search-row
+        %ul.cads-header__links.js-cads-copy-into-nav
+          - links.each do |link|
+            %li.cads-header__links-item
+              - if link.current_site?
+                %span.cads-header__text= link.title
+              - else
+                = link_to link.title, link.url, class: "cads-header__hyperlink"
+
+          - if custom_account_link.present?
```

Comment:
naming: how about `sign_out`? (this will be the slot that custom sign out html goes into?)

Reply:
Managed to rework this so there's a single slot for the account link now (the slot can take either arguments or a custom block). Keen to keep it general as it handles both cases. Think `account_link` feels right, or maybe `sign_in_out` to cover both cases?